tests/test_fileIO.py

```diff
@@ -20,9 +20,13 @@
 a_numpy_dict = {"a": a_numpy_array}
 
 
+@parameterized_class(
+    [
+        {"N_PROCS": 1},
+        {"N_PROCS": 2},
```

Comment:
Can you add a test for when the file does not exist?

Reply:
Looks like some of the tests are failing specifically because the file can't be found, so we need to bump a fix on that anyway